quick-cd.sh

```diff
@@ -10,6 +10,9 @@
 # (at your option) any later version.
 #
 
+# Disable POSIX-mode to relax constraints for func names.
+set +o posix
```

Comment:
Better to change the function name and fix the `smart-prompt.inputrc`... Anyway the user never calls it "manually" (by typing its name).